### ISSUE TITLE
Show reflected message from MUC

### ISSUE THREAD

**Issue by Zash** [dino/dino#590]
When sending a message to a MUC where the server rewrites the message in any way, Dino will only show the message as it was sent, not as it was received. If one has e.g. a MUC with a [built-in automatic pastebin for overly long messages](https://modules.prosody.im/mod_pastebin) then the final form of the message is not show.
It would be nice if the version of the message that was sent was replaced when receiving it back from the MUC.

**Comment by Zash** [comment]
This should probably rather be considered a bug.

Messes up multi-line messages sent to Biboumi (IRC gateway), which splits such messages into many and reflects them back, whereas Dino shows the original followed by duplicated lines from it.

**Comment by albjeremias** [comment]
is there a Pull Request or a way to understand this bug? It won't happen in conversations..

**Comment by Zash** [comment]
You send a

> very
> long
> message
> that
> triggers
> mod_pastebin

and the server modifies your message and sends it back like

> https://paste.example.com/72616e646f6d0a

but you stil see 

> very
> long
> message
> that
> triggers
> mod_pastebin

**Comment by licaon-kter** [comment]
Yup seeing the same for IRC via Biboumi on multiple lines messages. Conversations is smart and keeps one message only.
